tests/data/multi/train.forms-cs.txt

```diff
@@ -0,0 +1,100 @@
+Jednašedesátiletý Pierre Vinken se připojí ke správní radě jako nevýkonný ředitel dne 29 . listopadu .
```

Comment:
Ideally, we should add a README file describing the data (source, licence, preprocessing).